bangazonapi/views/lineitem.py

```diff
@@ -76,7 +79,8 @@ def destroy(self, request, pk=None):
         """
         try:
             customer = Customer.objects.get(user=request.auth.user)
-            order_product = OrderProduct.objects.get(pk=pk, order__customer=customer)
+            order_product = OrderProduct.objects.get(
```

Comment:
Your destroy method isn't working just yet. Take another look at your query to make sure you are getting the right thing. HINT: The pk from the client does not correspond to the product. Also, make sure you finish the destroy method ..  right now it's missing something pretty important!